actions/user_data.go

```diff
@@ -17,6 +17,7 @@ func userDataHandler(c buffalo.Context) error {
 
 	return c.Render(200, render.JSON(map[string]interface{}{
 		"id": user.ID,
+		"token": user.AuthToken,
```

Comment:
We don't want to return the auth token here because any valid user can use this on another user. So someone could use this to get the auth token of another user and do w/e I want.